packages/cli-kit/src/environment/local.test.ts

```diff
@@ -178,7 +178,7 @@ describe('cloudEnvironment', () => {
     ['GITPOD_WORKSPACE_URL', 'gitpod'],
   ])('returns correct cloud platform when %s is truthy', (envVar, platform) => {
```

Comment:
```suggestion
  ])('returns correct cloud platform when %s exists', (envVar, platform) => {
```

Reply:
`isTruthy` validates possible values that matches a boolean value. In this case, the values are homogeneous, so checking their existence is enough to consider them as a cloud environment

Reply:
Actually, do you think this works for all systems? `GITPOD_WORKSPACE_URL` will provide a URL, not a truthy value...

Reply:
It is working for the three cloud environments cli are currently supporting:
* gitpod
* codespaces
* spin
It's only checking that the environment variable exists. Maybe we could add a different validation depending on the value type

Reply:
ah yeah, sorry i misread